data-es2016plus.js

```diff
@@ -5714,6 +5714,36 @@ exports.tests = [
       }
     ]
   },
+  {
+    name: 'Ergonomic brand checks for private fields',
+    category: '2022 features',
+    significance: 'small',
+    spec: 'https://github.com/tc39/proposal-private-fields-in-in',
+    exec: function () {/*
+      class A {
+        #x;
+        static check(obj) {
+          return #x in obj;
+      rhino1_7_13: false
```

Comment:
Whops, this is wrong (it was also there before this PR)

```suggestion
```

Reply:
nice catch; it'd have been a noop at least, so it wouldn't break anything.

Reply:
Oh right, I didn't even realize it's a valid labeled statement.